client/src/api/user/requests.ts

```diff
@@ -0,0 +1,46 @@
+import { AxiosResponse } from 'axios';
+
+import { ErrorWrapper } from '../types';
+import { get, put } from '../builders';
+
+import * as Types from './types';
+
+const USER_ENDPOINT = '/users';
+
+// Returns all of the users in the database
+const getUsers = async (): Promise<
+  AxiosResponse<Types.GetUsersResponseType> | ErrorWrapper
+> => await get(USER_ENDPOINT, 'GET_USERS_FAIL');
+
+// Returns the logged in user
+const getCurrentUser = async (): Promise<
+  AxiosResponse<Types.GetCurrentUserResponseType> | ErrorWrapper
+> => await get('/auth/currentUser', 'GET_CURRENT_USER_FAIL');
+
+// Returns a single user from the user's id
+const getUser = async (
+  userId: string,
+): Promise<AxiosResponse<Types.GetUserResponseType> | ErrorWrapper> =>
+  await get(`${USER_ENDPOINT}/${userId}`, 'GET_USER_FAIL');
+
+// Adds a pitch to a user's claimed pitches
+const claimPitch = async (
```

Comment:
I get that the old name was confusing but should each function name start with 'get'/'create'/'delete'/'update' by convention? Maybe if you can think of a name that start with 'update' like 'updateAUsersClaimedPitches'?  i d k tho

Reply:
Didn't even think about this, thank you v much